modules/core/src/test/scala/gem/enum/Arbitraries.scala

```diff
@@ -23,6 +23,29 @@ trait Arbitraries {
   implicit val arbF2WindowCover = pick(F2WindowCover.all)
 
 
+  // Gmos
+
+  implicit val arbGmosAdc             = pick(GmosAdc.all            )
```

Comment:
I just wanted a shortcut to avoid repeating `Arbitrary(oneOf(...))` multiple times.  Maybe there's a better way though.

```scala
  private def pick[A](as: List[A]): Arbitrary[A] =
    Arbitrary(oneOf(as))
```